### ISSUE TITLE
DOC: stellar atmosphere libraries page

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#363]
It would be good to have a page documenting which stellar atmosphere libraries we have an some idea of when to use them.  Some appropriate plots would be good as well.

**Comment by lea-hagen** [comment]
Looks like a start to that is here:
https://beast.readthedocs.io/en/latest/beast_grid_inputs.html

**Comment by karllark** [comment]
I agree.  Forgot about that page.  Would be good to have some plots on the page showing the coverage in physical space (logT, logg, Z).
